hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/states/Node2ObjectsMap.java

```diff
@@ -83,7 +83,7 @@ public void insertNewDatanode(UUID datanodeID, Set<T> containerIDs)
    *
    * @param datanodeID - Datanode ID.
    */
-  void removeDatanode(UUID datanodeID) {
+  public void removeDatanode(UUID datanodeID) {
```

Comment:
Should annotate these methods as @VisibleForTesting

Reply:
Sure. Updated.